source/JsMaterialX/helpers.h

```diff
@@ -347,4 +346,17 @@ API(JSNAME, ems::optional_override([](SELF) { \
 #define BIND_0_0(...) \
 BIND_0(__VA_ARGS__)
 
+#define BIND_VALIDATE(CLASSNAME)                                                        \
+  .function("validate", ems::optional_override([](CLASSNAME &self) {                    \
+    return self.CLASSNAME::validate();                                                  \
```

Comment:
Do we need to call `self.CLASSNAME::validate()`? Why not simply `self.validate()`?